src/modules/integrator-guide/pages/accessing-corteza/ql.adoc

```diff
@@ -100,6 +100,20 @@ Filters for multi-value fields look precisely the same as they would for regular
 | selects = 'a2'
 | The filter returns the records where either one of the multi-value select options equals `a2`.
 
+// [#query-mv-example-6]#<<query-mv-example-6, -> do we need this part?
+// If so does it point to an example?
+| [#query-mv-example-6]#<<query-mv-example-6,Filter for concrete users>>#
+| users IN (A, B)
+| The filter returns record/s where either one of the multi-value select options contains `(A, B)`.
+
+| [#query-mv-example-7]#<<query-mv-example-7,Filter for concrete users>>#
+| users IN (SELECT id, firstName FROM employee WHERE id > 1)
```

Comment:
This query won't work? We don't support sub queries

Reply:
Removing this example